awx/main/tasks.py

```diff
@@ -256,7 +256,7 @@ def apply_cluster_membership_policies():
                 if instances_to_remove:
                     logger.info('Removing instances {} from group {}'.format(list(instances_to_remove), g.obj.name))
                     g.obj.instances.remove(*instances_to_remove)
-        logger.info('Cluster policy computation finished in {} seconds'.format(time.time() - started_compute))
+        logger.debug('Cluster policy computation finished in {} seconds'.format(time.time() - started_compute))
```

Comment:
I'm fine with switching this to debug, but you still have other `O(N)` info logs in this method. Since this really is a fairly extra-ordinary case, I would favor having exactly 1 INFO level log for the method, and then the others at debug, particular the ones that will emit multiple times in a single run.

Reply:
Preferably the single INFO level log would contain some kind of genuine but high-level information like "instances were added". If it's a no-op, then DEBUG is fine with me.